src/js/containers/search/filters/cfda/CFDASearchContainer.jsx

```diff
@@ -67,7 +66,6 @@ export class CFDASearchContainer extends React.Component {
 CFDASearchContainer.propTypes = propTypes;
 
 export default connect(
-    (state) => ({
-        selectedCFDA: state.filters.selectedCFDA }),
+    (state) => ({ selectedCFDA: state.filters.selectedCFDA }),
```

Comment:
New line for the body of this?

Reply:
We needed to either add a new line before the original `})` or remove the new line before `selectedCFDA`.
https://eslint.org/docs/rules/object-curly-newline